gemstone/Makefile

```diff
@@ -2,6 +2,7 @@ LIB_NAME := gemstone
 STATIC_LIB_NAME := lib$(LIB_NAME).a
 UDL_NAME := $(LIB_NAME).udl
 BUILD_MODE :=
+IPHONEOS_DEPLOYMENT_TARGET := 16.0
```

Comment:
let's set to `17.0` by default

Reply:
this is core lib, will set it on iOS repo